src/System.Net.Security/src/System/Net/Security/SslAuthenticationOptions.cs

```diff
@@ -66,6 +67,7 @@ internal SslAuthenticationOptions(SslServerAuthenticationOptions sslServerAuthen
         internal bool CheckCertName { get; set; }
         internal RemoteCertValidationCallback CertValidationDelegate { get; set; }
         internal LocalCertSelectionCallback CertSelectionDelegate { get; set; }
+        internal ServerCertCallback ServerCertSelectionDelegate { get; set; }
```

Comment:
'ServerCertCallback' => "ServerCertSelectionCallback" to be consistent with naming.

Reply:
Missed this one?